web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -18,12 +18,18 @@ use Drupal\Core\Url;
  *   Theme variables.
  */
 function dpl_react_apps_preprocess_page(array &$variables): void {
-  $search_result_url = Url::fromRoute('dpl_react_apps.search_result')->toString();
+  $options = ['context' => 'Search Header'];
+
   $data = [
     'search-header' => [
-      'search-header-url' => $search_result_url,
-      'alt-text' => t('Search field', [], ['context' => 'Search Header']),
-      'input-placeholder' => t('Start typing in order to search', [], ['context' => 'Search Header']),
+      'search-url' => DplReactAppsController::searchResultUrl(),
+      'material-url' => DplReactAppsController::materialUrl(),
+      'alt-text' => t('Search field', [], $options),
```

Comment:
Should be `$this-t. You can either change it or it will be fixed when merged with this pr: https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/122